README.md

```diff
@@ -1,3 +1,37 @@
 # KubeRobo
 
+KubeRobo will systematically demonstrate SIG Robotics’ end-to-end cloud-native robotics solution. It will serve as a comprehensive guide to all open source projects under the SIG Robotics umbrella, covering their scope, roadmap, and feature plans.
 
+![Introduce](./docs/images/intro.png)
+
+## Basic
+
+KubeEdge and its ecological components, such as sedna, ianvs, and edgemesh, serve as the technical base for a device-edge-cloud integrated robot solution, managing node equipment, orchestrating applications, network communications, and AI plugins.
+
+## Platform
+
+RoboOMS(Robot Operation Management Service) centrally manages multi-brand heterogeneous robots, supporting robot motion planning, control, and monitoring operation and maintenance.
+
+## Mapping
+
+RoboMAP(Robot Mapping) builds high-precision maps based on real scenarios to empower robot operation and simulation.
+
+## Simulation
+
+RoboSIM(Robot Simulation) simulates and verifies robot capabilities based on customer-defined scenarios.
+
+## Devtool
+
+RoboDev provides developers with a consistent development experience on the device and cloud, enabling fast and flexible robot application development.
+
+## Skill
+
+RoboSkill researches and develops robot applications to empower robots with intelligent movement, perception, decision-making, and execution.
+
+## Protocol (Will be drawn in the image later)
+
+The end-cloud universal robot control interface protocol facilitates robot cluster scheduling. Currently, it stipulates the interface model between AGV (which will be expanded to ubiquitous robots in the future) and the cloud control platform, client communication security certification, communication protocol structure, as well as the technical requirements and inspection rules for the control and transmission process interface.
+
+## Multi-Agent (Will be drawn in the image later)
+
+With the popularity of LLM and embodied AI, complex tasks can be completed by assigning role and task information to different agents and equipping them with corresponding tool plug-ins. The above functions can be converted into specific AI Agents, such as Coder (robot code producer), RoboManager (robot manager, management includes real-world robots and virtual robots), SimBuilder (simulation environment creator), Simulator (simulation parallel testing) member), Orchestrator (task organizer), Robot (similar to humans).
```

Comment:
It is not because of the popularity of "LLM and embodied AI" that we propose this, but we do recognize that large models (no only language model) bring `hope` to embodied intelligence, and kubeedge can provide support for this `hope`.

Reply:
Yes, that make sense.